x-pack/legacy/plugins/apm/public/context/LicenseContext/index.tsx

```diff
@@ -19,6 +19,10 @@ export function LicenseProvider({ children }: { children: React.ReactChild }) {
   const license = useObservable(license$);
   const hasInvalidLicense = !license?.isActive;
 
+  if (!license) {
+    return null;
+  }
```

Comment:
This adds an additional (third) case. I don't think that is necessary. The two states that's needed imo:

1. If the license has loaded AND is invalid, show `<InvalidLicenseNotification />`
2. If license is not loaded OR loaded and valid, show children: `<LicenseContext.Provider value={license} children={children} />`